src/ol/interaction/drawinteraction.js

```diff
@@ -618,6 +618,35 @@ ol.interaction.Draw.prototype.addToDrawing_ = function(event) {
 
 
 /**
+ * Remove last drawed point of the currently edited feature.
```

Comment:
I'd say 'Remove last point of the feature currently being drawn.'
Or perhaps 'Remove last drawn point.' or 'Cancel last point drawn.'